portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase

```diff
@@ -10,7 +10,7 @@ definition {
 	property testray.main.component.name = "Upgrades Security";
 
 	setUp {
-		SignIn.signIn();
+		User.firstLoginUI();
```

Comment:
No need to use **User.firstLoginUI**. Prefer to remove the url assertion from [line 15-17.](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase#L15-L17).

Reply:
Okay, I remove the assertion.